src/main/flight/imu.c

```diff
@@ -91,8 +91,8 @@ static float magneticDeclination = 0.0f;       // calculated at startup from con
 
 static imuRuntimeConfig_t imuRuntimeConfig;
 
-STATIC_UNIT_TESTED float q0 = 1.0f, q1 = 0.0f, q2 = 0.0f, q3 = 0.0f;    // quaternion of sensor frame relative to earth frame
-STATIC_UNIT_TESTED float rMat[3][3];
+float q0 = 1.0f, q1 = 0.0f, q2 = 0.0f, q3 = 0.0f;    // quaternion of sensor frame relative to earth frame
+float rMat[3][3];
```

Comment:
Rather than make these variables non-static, it would be better to make `HeadfreeEarthToBody` a function in `imu.c`, and call it `imuTransformVectorEarthToBody`.

Additionally add a function `imuRebaseEarthToBody` which sets q0=1 and q1,q2,q3=0.